src/Features/CSharp/Portable/UseExpressionBody/UseExpressionBodyCodeRefactoringProvider.cs

```diff
@@ -125,28 +124,26 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
             var succeeded = false;
             if (helper.CanOfferUseExpressionBody(preference, declaration, forAnalyzer: false))
             {
-                var title = s_helperToTitleMap[(helper, useExpressionBody: true)];
                 context.RegisterRefactoring(
                     CodeAction.Create(
-                        title,
+                        helper.UseExpressionBodyTitle.ToString(),
                         c => UpdateDocumentAsync(
                             document, root, declaration, helper,
                             useExpressionBody: true, cancellationToken: c),
-                        title),
+                        s_equivalenceKeyMap[(helper, useExpressionBody: true)]),
                     declaration.Span);
                 succeeded = true;
             }
 
             if (helper.CanOfferUseBlockBody(preference, declaration, forAnalyzer: false, out _, out _))
             {
-                var title = s_helperToTitleMap[(helper, useExpressionBody: false)];
                 context.RegisterRefactoring(
                     CodeAction.Create(
-                        title,
+                        helper.UseBlockBodyTitle.ToString(),
                         c => UpdateDocumentAsync(
                             document, root, declaration, helper,
                             useExpressionBody: false, cancellationToken: c),
-                        title),
+                        s_equivalenceKeyMap[(helper, useExpressionBody: false)]),
```

Comment:
I think such a pattern is used by many analyzers. @Youssef1313 did clean up bunch of these, but quite a few still exist in our code base.